pkgs/altera-quartus/generic.nix

```diff
@@ -369,7 +369,7 @@ stdenv.mkDerivation rec {
   name = "${baseName}-${version}";
   # version and srcs are unused by this derivation, but keep them as metadata
   # (for users).
-  inherit (quartusUnwrapped) version srcs;
+  inherit (quartusUnwrapped) version;
```

Comment:
I cannot reproduce it. The final (wrapped) quartus does not seem to depend on the installer files. I inspected the output of ```nix-store -qR $(nix-build pkgs/ -A altera-quartuses.altera-quartus-prime-lite)```.

Test case:

```
$ nix-build pkgs/ -A altera-quartuses.altera-quartus-prime-lite -o app >/dev/null && nix-store -qR ./app | grep "$(nix-build pkgs/ -A altera-quartuses.altera-quartus-prime-lite.srcs)"
(if there was some output here, it would mean the app depended on one of the sources)
```

Reply:
Ah, perhaps you mean build time dependency? I rely on the final quartus package to have src(s) attribute for build traceability, so I'd rather not remove that. Hm, perhaps it would be better with ```passthru = { inherit srcs; }```?

I don't see much point in removing that dependency though, if you build quartus you probably have just built quartus-unwrapped and have the sources already.